README.md

```diff
@@ -3,9 +3,14 @@ Nakadi Klients
 
 Implementation of a client accessing the low level API of the [Nakadi event bus](https://github.com/zalando/nakadi).
 
-## Prerequisites
-- Java >= 1.8
-- SBT >= 1.0
-- Scala >= 2.11.x
+<!-- Note: Currently the link you give above defaults to 'nakadi-jvm' branch, which is not what this code
+        works against. AKa270116
+-->
 
+## Prerequisites
+- JDK 8
+- sbt
```

Comment:
There is no sbt version 1.0. It's at 0.13.9 at the moment (this project uses 0.13.8).